tests/integration/helpers.py

```diff
@@ -33,7 +33,7 @@
 
 MODEL_CONFIG = {
     "logging-config": "<root>=INFO;unit=DEBUG",
-    "update-status-hook-interval": "1m",
+    "update-status-hook-interval": "29s",
```

Comment:
I would argue we should rather increase the interval, so that we don't hide charm issues by cleanup logic of update_status. And for the tests where we test the logic of this hook we decrease it as you did